docs/contribution.md

```
@@ -0,0 +1,43 @@
+  ```
+
+## 4. Pull Requests
+1. Run through the checklist in `.github/CODE_REVIEW_TEMPLATE.md`.  
+2. Use `RESOLVE` in comments when addressing feedback.  
+3. One PR = one issue/feature.
+
+## 5. Code Comments
+- **No** unnecessary comments—don’t explain what obvious code does.
+- For unfinished work, use `// TODO: what’s left to implement, or missing future implementation`.
+
+## 6. Deprecated Code
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)